src/pages/components/dropdown/usage.mdx

```diff
@@ -324,6 +326,22 @@ style of dropdown you are using.
 - Never use decorative images or icons within a dropdown.
 - We recommend presenting the options in alphabetical order.
 
+#### Parent checkbox
+
+- Since the parent checkbox is one among the options, do not describe it as an
+  action
+- Use the word “All” to describe this parent checkbox.
+- Alternatively you can add a parent word along with the word All to describe
+  the options in the menu.
+
+<Row>
+<Column colLg={8}>
```

Comment:
```suggestion
<Column colLg={12}>
```

I believe you intended this to be 12 columns instead of 8 by looking at your Figma file.

Reply:
changed to 12 columns